pom.xml

```diff
@@ -40,7 +40,7 @@
 
     <properties>
         <deploymentPath>/</deploymentPath>
-        <graph.core>1.1.20</graph.core>
+        <graph.core>1.1.30-SNAPSHOT</graph.core>
```

Comment:
https://cwiki.apache.org/confluence/display/MAVENOLD/Versioning
should be 21 :)

Reply:
ok